src/components/Txt/index.tsx

```diff
@@ -48,24 +60,54 @@ export const align = style({
 	cssProperty: 'text-align',
 });
 
+export const Lopy = (props: ThemedTxtProps) =>
+	props.truncate &&
+	css`
+		cursor: pointer;
+	`;
+
+export const LopyWrapper = styled.span`
+	display: inline-block;
+`;
+
 const BaseTxt = styled.div<TxtProps>`
 	${align}
 	${monospace}
 	${whitespace}
-
+	${code}
 	${caps}
 	${bold}
 	${italic}
-
 	${truncate}
 `;
 
 const Factory = (tag?: string) => {
 	return asRendition<React.FunctionComponent<TxtProps>>((props: any) => {
-		return <BaseTxt as={tag} {...props} />;
+		return props.copy ? (
+			<Copy
+				content={props.children}
+				show={props.hoverCopy ? 'hover' : 'always'}
```

Comment:
if we keep the same property as suggested in another comment

```suggestion
				show={props.showCopyMode || 'always'}
```